src/bazel/pkg_config_repository.bzl

```diff
@@ -86,6 +86,10 @@ def _exec_pkg_config(repo_ctx, flag):
         print("pkg-config is not found")  # buildifier: disable=print
         return []
     result = repo_ctx.execute([binary, flag] + repo_ctx.attr.packages)
+    # If result.stdout.strip() is empty, pkg-config will be re-executed with the --keep-system-cflags option added.
```

Comment:
Can you run [`buildifier`](https://github.com/bazelbuild/buildtools/blob/main/buildifier/README.md) to apply the standard format?

Otherwise, you would see CI error as follow.
https://github.com/phoepsilonix/mozc/actions/runs/11536533465/job/32112886990

Reply:
I see.

Reply:
```sh
go install github.com/bazelbuild/buildtools/buildifier@latest
$(go env GOPATH)/bin/buildifier --mode=diff --diff_command="diff -u" -r .
```
output is nothing.